charts/invenio/templates/_helpers.tpl

```diff
@@ -103,6 +103,16 @@
   {{- end }}
 {{- end -}}
 
+##########################     RabbitMQ protocol     ##########################
+{{/*
+  This template renders the protocol for RabbitMQ.
+*/}}
+{{- define "invenio.rabbitmq.protocol" -}}
+  {{- if .Values.rabbitmq.enabled }}amqp{{- else }}
```

Comment:
Please use the same style as elsewhere in the code. Example: https://github.com/inveniosoftware/helm-invenio/blob/2e2b043963edb0e37c507996419c195ddb21f1c1/charts/invenio/templates/_helpers.tpl#L70-L80